### DISCUSSION TITLE
Data export for Research Project

### DISCUSSION THREAD

**Discussion by ruedi99ms** [spatialsys/spatial-unity-sdk#22] [Help] [answered]
For a bigger research project, we want to use Spatial as our collaboration platform. Is it possible to asynchronously store or export data (e.g., collected from surveys (implemented as GUI))?

On the one hand, we will have live E-Townhall-Sessions with a group of people. There, we can do the data collection with Google Forms or the like. On the other hand, we want to implement some kind of treasure hunt (for gamification purposes of our research). Therefore, we need the collected data to be stored/exported without the researchers being online.

**Comment by AlanSpatial** [comment, marked as answer]
You can't export any data from inside spatial at this time. You can link people off platform to fill out a survey or something though.

**Reply by ruedi99ms** [reply]
Alright, thanks for your answer :-)
